src/main/java/Duke.java

```diff
@@ -19,10 +19,16 @@ public static void main(String[] args) {
                 break;
             } else if (next.equals("list")) {
                 for (int i = 0; i < counter; i++) {
-                System.out.printf("%d. %s\n",i +1,tasks[i]);
+                System.out.printf("%d.[%s] %s\n",i +1,tasks[i].getStatusIcon(),tasks[i].getDescription());
                 }
+            } else if (next.startsWith("done")){
+                String temp = next.replaceAll("\\D+","");
```

Comment:
This line is hard to read for me. It would help to use a more descriptive variable name other than temp and also indicate somewhere what you are doing here. I'm not sure what "\\D+" is